packages/circuit-ui/components/ColorInput/ColorInput.tsx

```diff
@@ -106,8 +105,35 @@ export const ColorInput = forwardRef<InputElement, ColorInputProps>(
 
     const hasSuffix = Boolean(suffix);
 
+    const handlePaste: ClipboardEventHandler<InputElement> = (e) => {
+      if (
+        !colorPickerRef.current ||
+        !colorInputRef.current ||
+        disabled ||
+        readOnly
```

Comment:
Is pasting even possible when the input element is disabled or read-only? I would expect browsers to prevent this

Reply:
With `readOnly` I could still paste, so leaving that one here. With disabled I can't focus the field so paste doesn't work, removing that one.